lib/script.go

```diff
@@ -0,0 +1,101 @@
+package lib
+
+import (
+	_ "embed"
+	json2 "encoding/json"
+	"errors"
+	"fmt"
+	schema "github.com/xeipuuv/gojsonschema"
+	"io/ioutil"
+	"regexp"
+	"sort"
+	"strings"
+	"time"
+)
+
+//go:embed "script_schema.json"
+var SCRIPT_SCHEMA []byte
```

Comment:
Go variables don't typically come in capital snake case. Camel case would be preferred

```suggestion
var ScriptSchema []byte
```